html/om_oss.html

```diff
@@ -1,6 +1,5 @@
 <!DOCTYPE html>
 <html lang="sv">
-
 <head>
```

Comment:
We need to add a meta description in `<head>`.
Something like "Information om Konstnärsnämnden" or something :)